### DISCUSSION TITLE
Einlesen der CSV Datei

### DISCUSSION THREAD

**Discussion by MehmetCagferoglu** [irgroup-classrooms/dis17-2021#9] [Q&A]
Hallo Zusammen, 
bei deM Tutorial in Aufgabe 2 müssen wir die CSV Datei einlesen. 

C:\solr-8.10.0> java -jar -Dc=films -Dparams=f.genre.split=true&f.directed_by.split=true&f.genre.separator=|&f.directed_by.separator=| -Dauto example\exampledocs\post.jar example\films\*.csv

Doch wir kriegen als Fehlermeldung 

"&" kann syntaktisch an dieser Stelle nicht verarbeitet werden.

Bitte um Hilfe!
Danke

**Comment by xxjuly** [comment]
Ich hab die Params in doppelte Anführungszeichen gesetzt und danach funktionierte es:  

```java -jar -Dc=films -Dparams="f.genre.split=true&f.directed_by.split=true&f.genre.separator=|&f.directed_by.separator=|" -Dauto example\exampledocs\post.jar example\films\*.csv```

**Reply by MehmetCagferoglu** [reply]
Sorry für die späte Antwort, hat geklappt! Danke dir @xxjuly
